extensions/amp-story-interactive/0.1/amp-story-interactive-abstract.js

```diff
@@ -363,6 +368,14 @@ export class AmpStoryInteractive extends AMP.BaseElement {
 
   /** @override */
   layoutCallback() {
+    if (
+      isExperimentOn(this.win, 'amp-story-interactive-disclaimer') &&
+      this.element.hasAttribute('endpoint')
+    ) {
+      // Needs to be called after buildCallback to measure properly.
+      this.disclaimerEl_ = buildInteractiveDisclaimer(this);
+      this.rootEl_.appendChild(this.disclaimerEl_);
```

Comment:
a11y ux:
On a screen reader this would be read after reading all of the quiz content.
Using `.prepend` can help ensure a screen reader will read this before the quiz content.

You may need to add `z-index: 1;` to the disclaimer CSS since this will change the layout order.